### PR TITLE
Feature sort v3

### LINKED ISSUE #144: Feature Sort v3

### PR DESCRIPTION
**Pull request by kumsssss**
1) Changed the implementation of SortCommandParser so that extra spaces between words will not throw error.

Closes #144

### CODE REVIEW

File: src/main/java/seedu/storemando/logic/parser/SortCommandParser.java [hunk 1]
```diff
@@ -21,9 +22,13 @@ public SortCommand parse(String args) throws ParseException {
         String trimmedArgs = args.trim();
         if (trimmedArgs.equalsIgnoreCase(EXPIRYDATE_KEYWORD)) {
             return new SortExpiryDateCommand();
-        } else if (trimmedArgs.equalsIgnoreCase(QUANTITY_ASCENDING_KEYWORD)) {
+        }
+        String[] splitWords = trimmedArgs.split("\\s+");
+        if (splitWords.length == 2 && splitWords[0].equalsIgnoreCase(QUANTITY_KEYWORD)
+            && splitWords[1].equalsIgnoreCase(ASCENDING_KEYWORD)) {
             return new SortQuantityCommand(true);
```
Comment (chewwh09) [inline src/main/java/seedu/storemando/logic/parser/SortCommandParser.java]: Prevent using magic boolean. Other than that LGTM

**Review by chewwh09 — APPROVED** [review]
Just nitpicking 1 part of your code :D Other than that LGTM!

File: src/main/java/seedu/storemando/logic/parser/SortCommandParser.java (hunk 1 above)
Comment (JayChenYJ) [inline src/main/java/seedu/storemando/logic/parser/SortCommandParser.java]: I think SortQuantityCommand shouldn't be taking in a boolean :)
1 suggestion I can think of is to create 2 separate classes handling asc and desc. I think there is a more elegant way though :)

Comment (kumsssss) [reply src/main/java/seedu/storemando/logic/parser/SortCommandParser.java]: alright will do so!

Comment (kumsssss) [reply src/main/java/seedu/storemando/logic/parser/SortCommandParser.java]: ok will make the changes

**Pull request merged**